### PR TITLE
Turn ha-cards.html into ES6 class

### PR DESCRIPTION
**Pull request by andrey-git**
Turn `ha-cards.html` into ES6 class

### CODE REVIEW

File: src/components/ha-cards.html [hunk 1]
```diff
@@ -95,7 +92,7 @@
   // groups: X
   // rest: 100
 
-  var PRIORITY = {
+  const PRIORITY = {
```
Comment (NovapaX) [inline src/components/ha-cards.html]: this change from 'var' to 'const' seems to break the interface on safari (both macOS and iOS)
`ReferenceError: Can't find variable: PRIORITY`

**Review by NovapaX — COMMENTED** [review]
breaking safari on macOS and iOS

**Comment by andrey-git** [comment]
Which safari version is this?

**Comment by andrey-git** [comment]
Are you running in dev mode?

**Comment by NovapaX** [comment]
Did some more research. `const` is block scoped.
calling `getPriority("sensor")` in the Console on below codepen works in chrome, fails in safari:
 https://codepen.io/anon/pen/rYxaxx?editors=1111

Running dev mode, safari 11.0. iOS 11.1

**Comment by andrey-git** [comment]
let / const is block scoped.
functions are not block scoped (so my PR made those functions public, and I should fix that)
What call is failing on Safari?

**Comment by NovapaX** [comment]
Deleting the outermost brackets within the <script> tag solves the issue (in dev mode at least). Don't know what consequences removing those brackets have.

The call to getPriority() fails from within ComputeCards() line 295

**Pull request merged**